torch/_logging/structured.py

```diff
@@ -0,0 +1,18 @@
+"""
+Utilities for converting data types into structured JSON for dumping.
+"""
+
+import traceback
+from typing import Sequence
+
+def from_traceback(tb: Sequence[traceback.FrameSummary]) -> object:
```

Comment:
```suggestion
def from_traceback(tb: Sequence[traceback.FrameSummary]) -> List[Dict[str, object]]:
```

Reply:
due to mutability of list/dict, this doesn't type check!